emailalerts/emailsystem.py

```diff
@@ -7,13 +7,19 @@
 from email.mime.multipart import MIMEMultipart
 from dashboard.alerts import alert
 
-email_config_file = open('/etc/capstone-ids/config.json', 'r')
-email_config_data = json.load(email_config_file)
+try:
+    email_config_file = open('/etc/capstone-ids/config.json', 'r')
```

Comment:
change filepath to '/etc/tinyHIPPO/config.json'

Reply:
Done